doc/guides/kconfig/index.rst

```diff
@@ -275,6 +275,85 @@ In summary, here are some recommended practices for ``select``:
   you like. They're a great tool for simplifying Kconfig files.
 
 
+(Lack of) conditional includes
+******************************
+
+``if`` blocks add dependencies to each item within the ``if``, as if via
```

Comment:
Minor thing: The "as if via" construct seems a bit difficult to read.
Maybe something like "as if depends on was used" would be a bit better?

Reply:
Yup, better. Changed.